### DISCUSSION TITLE
CamelCase package names?

### DISCUSSION THREAD

**Discussion by tgstaples** [pdm-project/pdm#2603] [Q&A] [answered]
Hello, does PDM support CamelCase package names?

**Comment by frostming** [comment]
I don't understand what you mean, did you encounter any issues?

**Reply by tgstaples** [reply]
Sorry for the confusion.  Just wondering if PDM requires package names to be all lowercase, like Poetry does?

**Reply by frostming** [reply, marked as answer]
No, it will be normalized internally when being looked for in the index.

**Reply by frostming** [reply]
Better to discuss on a phenomenon or problem, otherwise I don't know what you want to talk about.
